src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcRouteHandler.cs

```diff
@@ -45,25 +44,18 @@ public Task RouteAsync(RouteContext context)
 
             EnsureServices(context.HttpContext);
```

Comment:
Yeah, I'm getting around to that. Just wanted to send out what I have so you could start digging in.